src/pages/public_registration/PublicRegistration.tsx

```diff
@@ -16,9 +16,11 @@ const PublicRegistration: FC = () => {
   const user = useSelector((store: RootStore) => store.user);
 
   const isAllowedToSeePublicRegistration =
-    registration?.status === RegistrationStatus.PUBLISHED ||
-    (user?.isCurator && registration?.publisher.id === user?.customerId) ||
-    registration?.owner === user?.id;
+    registration &&
```

Comment:
Litt av en munnfull. Nesten så man kunne lagt sub-funkskjoner under denne igjen for bedre lesbarhet. f.ex IsOwner(...), isPublished(...) osv.

Reply:
lurt! 
https://github.com/BIBSYSDEV/NVA-Frontend/pull/1323#discussion_r546580494